mne/io/kit/coreg.py

```diff
@@ -167,12 +170,22 @@ def _set_dig_kit(mrk, elp, hsp, eeg):
                 "%s." % (elp, elp_points.shape)
             )
         elp = elp_points
-    elif len(elp) not in (6, 7, 8):
+        if len(bad_coils) > 0:
+            elp = np.delete(elp, np.add(bad_coils, 3), 0)
```

Comment:
Makes it explicit that it casts to `ndarray` then just uses standard `__add__`
```suggestion
            elp = np.delete(elp, np.array(bad_coils) + 3, 0)
```